scrooge-generator/src/test/scala/com/twitter/scrooge/ThriftStructMetaDataSpec.scala

```diff
@@ -162,7 +162,7 @@ class ThriftStructMetaDataSpec extends Spec {
     val bonk = MorePerfectUnion.Bonk(Bonk("message", 55))
     assert(Some(bonkFieldInfo) == bonk.unionStructFieldInfo)
 
-    bonk.containedValue match {
+    bonk.containedValue() match {
```

Comment:
This fails to compile because ```containedValue``` in ```MorePerfectUnion``` is parameterless and it's return type isn't actually ```Unit``` so this shouldn't change.